package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-eligibility",
-  "version": "0.16.0",
+  "version": "0.16.1",
```

Comment:
note: there may be version number conflict with previous PR

Reply:
sure, will bump mine to`0.16.2`